src/helpers.ts

```diff
@@ -64,6 +64,8 @@ import {
 } from '@ethersproject/transactions';
 import {getDerivationPath} from './hdpath';
 import {bnReplacer} from './internal/utils';
+import {DeploymentFactory} from './DeploymentFactory';
+import {link} from 'fs';
```

Comment:
I guess this was added by mistake ?

Reply:
Hmm interesting , might be a left over, will clean this up.

Reply:
Hello @rmeissner! Please complete required changes for this PR to be merged and published.
Thanks in advance!